test/partiql-tests-runner/README.md

```diff
@@ -21,3 +18,18 @@ tests in `lang` that were ported to `partiql-tests`
 PARTIQL_TESTS_DATA=/path/to/partiql-tests/data \
 ./gradlew :test:partiql-tests-runner:test --tests "*ConformanceTestsReportRunner" -PconformanceReport
 ```
+The report is written into file `test/partiql-tests-runner/conformance_test_results.ion`.
+
+## Run Conformance Tests in IntelliJ
+
+The above project property `-PconformanceReport` is checked in `test/partiql-tests-runner/build.gradle.kts`,
+to prevent the conformance test suite from executing during a normal project-build test run. 
+Unfortunately, this also disables running `ConformanceTestsReportRunner` via IntelliJ UI for unit tests. 
+To make that possible locally, temporarily comment out the check in `test/partiql-tests-runner/build.gradle.kts`.
```

Comment:
If there is a better way to run the conformance suite as unit tests in IntelliJ, let's write it up here. 

Otherwise, something better is desirable.  It might be possible to derive something suitable from https://docs.gradle.org/current/userguide/java_testing.html#sec:configuring_java_integration_tests

Reply:
The way I was running the conformance tests in `DefaultConformanceTestRunner` was running the specific parameterized test function (e.g. `validatePartiQLEvalTestData` and `validatePartiQLEvalEquivTestData`). This didn't require commenting out the following line. And let me run the tests with IntelliJ. Perhaps we could add a comment about this workaround.

Reply:
Running the entire class of tests though required me to comment out those lines.

Reply:
I tried to run the functions directly, but the exclude from `build.gradle.kts`, even though it's on the `ConformanceTestsReportRunner` *class*, applies to the member functions as well.  
Perhaps your experience was different because it was indeed for `DefaultConformanceTestRunner`, which was not excluded (till this PR, hopefully temporarily...)

Reply:
Ah, I see now. Ignore my followup comment then. I was running the functions without this PR's changes locally so my workaround doesn't actually work. For now, the only way to run the tests locally would be to comment out that `exclude` line in `build.gradle.kts`.